src/command.coffee

```diff
@@ -102,7 +102,17 @@ exports.run = ->
   process.argv = process.argv[0..1].concat literals
   process.argv[0] = 'coffee'
 
-  opts.output = path.resolve opts.output  if opts.output
+  if opts.output
+    outputBasename = path.basename opts.output
+    if '.' in outputBasename
```

Comment:
What about `--output dir/.` and `--output dir/sub/..`? I guess those two are supposed to mean `--output dir/` and `--output dir/`? Also, there's `--output .` and `--output ..`.

Also, a question: Is it possible to add a trailing slash to _force_ a directory? `--output dir/script.js/`